src/subscription_manager/gui/registergui.py

```diff
@@ -153,6 +171,11 @@ class RegisterInfo(ga_GObject.GObject):
     port = ga_GObject.property(type=str, default='')
     prefix = ga_GObject.property(type=str, default='')
 
+    server_info = ga_GObject.property(type=ga_GObject.TYPE_PYOBJECT, default=None)
+
+    # Used to control whether or not we should use the PerformUnregisterScreen
+    do_unregister = ga_GObject.property(type=bool, default=False)
```

Comment:
Hadn't noticed this before, but for Gtk widgets (and maybe GObjects?) a method name of 'do_blah_blah' is a little
special and magic. For a GtkWidget a method called 'do_some_signal_name' is kind of automatically connected to the 'some-signal-name' emission. 

In this case, do_unregister is a attribute/property, not a method, and that object doesn't have a 'unregister' signal, so I don't think it will cause problems, just a little confusing.